query-engine/driver-adapters/js/connector-test-kit-executor/src/index.ts

```diff
@@ -250,4 +250,14 @@ async function libsqlAdapter(url: string): Promise<DriverAdapter> {
     return new PrismaLibSQL(libsql)
 }
 
+function postgres_options(url: string): any {
+    let args: any = {connectionString: url}
+    const schemaName = new URL(url).searchParams.get('schema')
+    if (schemaName != null) {
+        args.options = `--search_path="${schemaName}"`
+    }
+    return args;
```

Comment:
Ugh, this ass `&option=--search_path="foo"` to the connection string, correct?
Is that something our users also have to do to be able to use raw queries properly?
Is that something that quaint does automatically on connect for the `schema` (or `public` if not set)?

Reply:
This is only in tests. This tells pg, to add search path on connect, yes. 

> Is that something that quaint does automatically on connect for the schema (or public if not set)?

Quaint users need not to do this, we do it in quaint by extracting from the connection string the schema name and setting ourself the params on connect.

> Is that something our users also have to do to be able to use raw queries properly?

Our users, as providers of the driver implementation (as per our UX) will have to set the driver accordingly, for raw queries to find the path. If they used the raw driver and issue a raw query they would also need to do the same thing. If they used prisma with rust, and move to pg, they will need to configure pg explicitly. The alternative as we are not in control of the driver, is that we issue a first query setting the search path, which was the original implementaiton in this PR, that I changed based on Flavian's feedback and my own judgement.

The tradeoff is:
- No query on start -> faster coldstarts but users need to configure the search path on their driver options.
- Query on start -> slower colstarts, better UX

I opted to the first case in this final state of this PR.

Reply:
Yeah we need to figure that out. I think you can just create an issue for this, and copy your comment here into it - and then we will revisit. Either we go to your first implementation, or we need to properly document that. I am currently not sure what it should be. (Prisma's `?schema` API is pretty unique and weird compared to other drivers - maybe an opportunity to get rid of that and replace it with something more `search_path` like.)

Reply:
Issue created: https://github.com/prisma/team-orm/issues/438